chapter2.adoc

```diff
@@ -12,8 +12,10 @@ that can be held in an M-mode CSR and modifiable by the RDSM via CSR r/w
 instructions  - herewith called the `supervisor domain identifier` assigned to
 the hart. Supervisor domains are orthogonal to hart privilege levels and since
 Smmtt enables physical memory isolation, there is one CSR (per hart) managed by
-M-mode. Device side accesses are addressed in the `IO-MTT` extension. Isolation
-of data within a device is out of scope of this specification.
+M-mode. `Smsdid` is expected to be used in conjunction with `Smmtt` for physical
+memory isolation along with alternate mechanisms such as `e(PMP)`. Device side
+accesses are addressed in the `IO-MTT` extension. Isolation of data within a
+device is out of scope of this specification.
```

Comment:
The text seems to imply that e(PMP) are alternates to MTT. Whereas MTT and e(PMP) are applicable all the time.

Reply:
I can't see the latest commits for some reason?

But I think I agree with Ved's reasoning. All physical checks are always there. MTT mode "bare" simply disables the MTT check (always passes).

Then we can leave use case discussions to the security model. We already have both with and without MTT there.

Nit - it should be (e)PMP, not e(PMP)? Or PMP/ePMP to be really clear... It could be PMA as well, or whatever. 

But all we need to say here is that all RISC-V physical memory checks are always performed, whether MTT is active or not.